pkg/skuba/actions/node/upgrade/apply.go

```diff
@@ -138,6 +141,16 @@ func Apply(client clientset.Interface, target *deployments.Target) error {
 	} else if err := target.Apply(nil, "kubeadm.upgrade.node"); err != nil {
 		return err
 	}
+	if nodeVersionInfoUpdate.HasMajorOrMinorUpdate() {
```

Comment:
Curious: wouldn't an upgrade of crio happen already when running the kubernetes.install-node-pattern above?

Reply:
That will install the transient pattern that won't include kubelet or crio.

Reply:
Awesome. I thought it was the definitive already, my bad. Good then!